.travis.yml

```diff
@@ -34,7 +34,7 @@ install:
   - pip install pytest-flake8
   - pip install portalocker
   # Dependencies not available through conda, install through pip
-  - pip install pysatCDF >/dev/null
+  - pip install cdflib==0.3.19
```

Comment:
Do we want this to be `>=`?

Reply:
It should be a cap.  But I think there are issues with the travis install if <= is used.

Reply:
Yeah, 0.3.20 breaks cdflib for pysatNASA at the moment and <= requires some extra characters in travis and I haven't tested any versions other than 0.3.19